src/components/ebay-table/table.stories.ts

```diff
@@ -112,6 +117,27 @@ export default {
                 category: "@row attribute tags",
             },
         },
+        a11yLoadingText: {
+            description: "Text for progress bar expressive when table is in loading body state",
```

Comment:
It would be helpful to say `Localized text`. Helps this needs to be localized

Reply:
I believe that should be understood from this. I think for each `a11y` text override its assumed that it's for localization. No need to have it called out for each one.